### PR TITLE
add support for conan dependency manager.

### PR DESCRIPTION
**Pull request by SylvestreG**
# Pull Request Template

## Description

add support for conan dependency manager.

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [X] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 18.10.x
- [ ] 19.04.x
- [ ] 19.10.x
- [X] 20.04.x (master)

#### Community contributors & Centreon team

- [X] I followed the **coding style guidelines** provided by Centreon
- [X] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [X] I have commented my code, especially **hard-to-understand areas** of the PR.
- [X] I have made corresponding changes to the **documentation**.
- [X] I have **rebased** my development branch on the base branch (master, maintenance).

#### Centreon team only

- [X] I have made sure that the **unit tests** related to the story are successful.
- [X] I have made sure that **unit tests cover 80%** of the code written for the story.
- [X] I have made sure that **acceptance tests** related to the story are successful (**local and CI**)

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -59,72 +78,8 @@ configure_file(
...
-link_directories(${binary_dir})
-include_directories(${source_dir})
-
-ExternalProject_Get_Property(spdlog_lib source_dir)
-
-include_directories(${source_dir}/include)
-
-ExternalProject_Get_Property(Asio source_dir)
-include_directories(${source_dir}/asio/include)
+include(ExternalProject)
```
Comment (bouda1) [inline CMakeLists.txt]: Always needed?

File: README.md [hunk 2]
```diff
@@ -69,11 +69,17 @@ This paragraph is only a quickstart guide for the compilation of
 Centreon Broker. For a more in-depth guide with build options you should
 refer to the [online documentation](https://documentation.centreon.com/docs/centreon-broker/en/latest/installation/index.html#using-sources).
 
+For the projet compilation you need to have conan installed. To install conan you need pip 
+(python package manager). You can install conan like that.
+
+    $> pip install conan
+
```
Comment (bouda1) [inline README.md]: There is also an alternative for others distributions than Centos7, isn't there?
Maybe a little explanation on that could be interesting.

File: neb/inc/com/centreon/engine/configuration/applier/logging.hh [hunk 3]
```diff
@@ -40,7 +40,8 @@ class logging {
  public:
   void apply(configuration::state& config);
   static logging& instance();
-  void clear();
+  static void load();
+  static void unload();
```
Comment (bouda1) [inline neb/inc/com/centreon/engine/configuration/applier/logging.hh]: Why this in the middle of all the makefiles? :smile: 
Leftover? I don't see.

File: CMakeLists.txt (hunk 1 above)
Comment (SylvestreG) [reply CMakeLists.txt]: good catch

**Pull request merged**